Update readme file (#337)
apache/pulsar-manager: README.md

    ```
    After find the ip address of the Pulsar Manager, you can access the Pulsar Manager at `http://${pulsar-manager-cluster-ip}/#/environments`.

### Build from bin package

```
wget https://dist.apache.org/repos/dist/release/pulsar/pulsar-manager/apache-pulsar-manager-0.2.0/apache-pulsar-manager-0.2.0-bin.tar.gz
tar -zxvf apache-pulsar-manager-0.2.0-bin.tar.gz
cd pulsar-manager
tar -zxvf pulsar-manager.tar
cd pulsar-manager
cp -r ../dist ui
./bin/pulsar-manager
```

> Now you can now access it at the following address: frontend => http://localhost:7750/ui/index.html.

Reviewer: ```suggestion
Now, you can access it at the following address: frontend => http://localhost:7750/ui/index.html.
```
it refers to Pulsar Manager? if yes, please use Pulsar Manager here.

> Note, now you can now access it at the following address: frontend => http://localhost:7750/ui/index.html

Reviewer: ```suggestion
Now, you can access Pulsar manager at the following address: frontend => http://localhost:7750/ui/index.html
```

> [Set the administrator account and password](#access-pulsar-manager).

Reviewer: ```suggestion
For details, see [Set the administrator account and password](#access-pulsar-manager).
```

> [Set the administrator account and password](#access-pulsar-manager)

Reviewer: ```suggestion
[Set the administrator account and password](#access-pulsar-manager).
```

> #### Enable bookkeeper visual manager(optional)

Reviewer: ```suggestion
#### Enable BookKeeper visual manager(optional)
```

> Update the configuration file `pulsar-manager/bkvm.conf`, set `bkvm.enabled` to true

Reviewer: ```suggestion
Update the configuration file `pulsar-manager/bkvm.conf`, and set `bkvm.enabled` to `true`.
```
